tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/server/StringLSP4J.java

```diff
@@ -241,10 +242,13 @@ public String toString(Command command) {
 		if (command == null) {
 			return "";
 		}
+		List<Object> argumentsRelativized = command.getArguments().stream()
+				.map(this::relativizeIfURIString)
+				.collect(Collectors.toList());
 		String str = Strings.join(", ",
 				command.getTitle(),
 				command.getCommand(),
-				Strings.toString(command.getArguments()));
+				Strings.toString(argumentsRelativized));
```

Comment:
That does not work because `this::relativizeIfURIString` is a function from Object to Object, not from Object to String as is expected by `String.toString()`.